app/helpers/taxonomy_helper.rb

```diff
@@ -11,6 +11,9 @@ def show_organization_tab?
   end
 
   def show_taxonomy_tabs?
+    Foreman::Deprecation.deprecation_warning(
+      '1.15', 'The partial "taxonomy/loc_org_tabs" checks already if '\
```

Comment:
1.13's the current (well, latest) stable, so this should be 1.16 (deprecated in 1.14, 1.15 and then is removed in 1.16).